Model/Site.hs

```diff
@@ -42,3 +46,20 @@ destroySite siteId = do
 
 overSiteQuota :: [a] -> Plan -> Bool
 overSiteQuota xs Plan{..} = length xs >= planSiteQuota
+
+siteBranded :: SiteId -> DB Bool
+siteBranded siteId = do
```

Comment:
Oh right. I had originally split it because I wanted to handle no members specially, but I'm not doing that any longer. Back down the esqueleto rabbit hole...